File: 1-easy/problems/3688-bitwise-or-of-even-numbers-in-an-array/solution.ts
function evenNumberBitwiseORs(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return reduce(lambda total, x: total|(x if x%2 == 0 else 0), nums, 0);
}

export { evenNumberBitwiseORs };

File: 1-easy/problems/3688-bitwise-or-of-even-numbers-in-an-array/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def evenNumberBitwiseORs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return reduce(lambda total, x: total|(x if x%2 == 0 else 0), nums, 0)
